src/postgres_mcp/server.py

```diff
@@ -14,238 +12,239 @@
 from .dta.safe_sql import SafeSqlDriver
 from .dta.sql_driver import SqlDriver
 
-server = Server("postgres-mcp")
+mcp = FastMCP("postgres")
```

Comment:
IMHO, let's keep using a distinguishable name for our product.
```suggestion
mcp = FastMCP("postgres-mcp")
```

Reply:
changed